Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -803,6 +803,16 @@
 			value="/WEB-INF/views/publicnetworkvisual.jsp" />
 	</definition>
 
+    <definition name="sites/projectblog" extends="base.definition.public">
+		<put-attribute name="currentPage" value="projectblog" />
+		<put-attribute name="content" value="/WEB-INF/views/projectblog.jsp" />
+	</definition>
+	
+	<definition name="sites/addprojectblog" extends="base.definition.public">
+		<put-attribute name="currentPage" value="addprojectblog" />
```

Comment:
what entry do you want to have active in the navigation bar? The value for this navigation bar entry should be here (check navbar.jsp).